src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -33,17 +31,23 @@
 import _get from 'lodash/get';
 import _map from 'lodash/map';
 import lendByCategoryQuery from '@/graphql/query/lendByCategory.graphql';
-import categoriesByIdQuery from '@/graphql/query/categoriesById.graphql';
+import loanChannelQuery from '@/graphql/query/loanChannelData.graphql';
 import WwwPage from '@/components/WwwFrame/WwwPage';
 import CategoryRow from '@/components/LoansByCategory/CategoryRow';
-import CategoryAdminControls from './CategoryAdminControls';
-import LoadingOverlay from './LoadingOverlay';
+
+// Parse a SettingsManager value
+function readSetting(data, key) {
+	try {
+		return JSON.parse(JSON.parse(_get(data, key)));
+	} catch (e) {
+		return null;
+	}
+}
 
```

Comment:
factor this out to be a utility function?

Reply:
My thoughts exactly! I'm planning on pulling that out into a settings manager utility that would also have a function for checking if a feature/experiment is enabled based on start/end time and the enabled flag.